### PR TITLE
EV-1285 disable job updater

### PR DESCRIPTION
**Pull request by erutherford**
# What
* Elroy has updated logic to handle watching jobs, we're going to rely on that updated logic and not update elroy when kit's determined it was successful or failed.

### CODE REVIEW

**Comment by jlordiales** [comment]
We should deploy the changes in Elroy before releasing this (just a note for me mostly)

File: Functional-Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+FROM node:6.11.5-alpine
```
Comment (c1freitas) [inline Functional-Dockerfile]: Scope Creep :P

Comment (erutherford) [reply Functional-Dockerfile]: It was mostly a copy and paste I think I just added the copy of test files

File: src/lib/manifests.js [hunk 2]
```diff
@@ -86,12 +86,13 @@ class Manifests extends EventEmitter {
   }
 
   /**
-   * Tests and returns whether or not the deployment contains a manifest
+   * Tests and returns whether or not the manifests includes a k8s resource on the elroy update blacklist
    * @param {array} generatedManifests - an array of manifests to check
    */
-  isStatefulSet(generatedManifests) {
+  inElroyUpdateBlacklist(generatedManifests) {
+    let elroyUpdateBlacklist = ["StatefulSet", "Job"];
```
Comment (jlordiales) [inline src/lib/manifests.js]: Minor: maybe this could come from the ENV, to make it easier to add extra resources in the future as we move individual types to the new watcher?

Comment (erutherford) [reply src/lib/manifests.js]: I'll make that change when/if we need to do it for CronJobs

Comment (erutherford) [reply src/lib/manifests.js]: I went ahead and added it, I'll get the plumbing in place in kit-server for this tomorrow morning.
